sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/trees/TreePatterns.scala

```diff
@@ -110,8 +105,28 @@ object TreePattern extends Enumeration  {
   val UP_CAST: Value = Value
   val DISTRIBUTED_SEQUENCE_ID: Value = Value
 
+  // Unresolved expression patterns (Alphabetically ordered)
+  val UNRESOLVED_ALIAS: Value = Value
+  val UNRESOLVED_ATTRIBUTE: Value = Value
+  val UNRESOLVED_COLLATION: Value = Value
+  val UNRESOLVED_DESERIALIZER: Value = Value
+  val UNRESOLVED_DF_STAR: Value = Value
+  val UNRESOLVED_HAVING: Value = Value
+  val UNRESOLVED_IDENTIFIER: Value = Value
+  val UNRESOLVED_ORDINAL: Value = Value
+  val UNRESOLVED_FUNCTION: Value = Value
+  val UNRESOLVED_HINT: Value = Value
+  val UNRESOLVED_WINDOW_EXPRESSION: Value = Value
+  val UNRESOLVED_PLAN_ID: Value = Value
```

Comment:
Let's sort this sub group alphabetically as well?

Reply:
+1